File: app/src/main/java/com/oplus/fwandroid/common/utils/ClipUtil.java
package com.oplus.fwandroid.common.utils;

import android.content.Context;

/**
 * Created by Sinaan on 2016/12/22.
 */
public class ClipUtil {
    /**
     * 实现文本复制功能
     *
     * @param content
     */
    public static void copy(Context context,String content) {
        SystemService.getClipboardManager(context).setText(content.trim());
    }
}

File: app/src/main/java/com/oplus/fwandroid/common/utils/ClipUtil.kt
package com.oplus.fwandroid.common.utils

import android.content.Context

/**
 * @author Sinaan
 * @date 2020/7/13
 * GitHub：https://github.com/wanganan
 * email：waa182838@sina.com
 * description：
 * version: 1.0
 */
object ClipUtil {
    /**
     * 实现文本复制功能
     *
     * @param content
     */
    fun copy(context: Context?, content: String) {
        SystemService.getClipboardManager(context!!)?.text = content.trim { it <= ' ' }
    }
}